test-keys/bk-server/id_rsa_sage_beekeeper

```diff
@@ -0,0 +1,49 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Do you want to commit this private key?

if so do we need to make the repo private?  or since this is just test keys do we not care?

Reply:
this is just a test key, it is public intentionally (but still thanks for asking, better safe than sorry)